src/main/ui/src/components/Login/Base.module.css

```diff
@@ -0,0 +1,17 @@
+.button_common {
```

Comment:
I think we could use the bootstrap button and it's built in errors here as well.